### PR TITLE
upgrade to katzenpost v0.0.27

### PR DESCRIPTION
**Pull request by mixmasala**

### CODE REVIEW

**Comment by mixmasala** [comment]
https://github.com/katzenpost/katzen/issues/42

File: Makefile [hunk 1]
```diff
@@ -74,7 +74,7 @@ docker-build-nix: docker-nix-base
 docker-alpine-base: $(go_package_cache_dir)
 	@if ! $(docker) images|grep katzen/alpine_base; then \
 		$(docker) run --replace --name katzen_alpine_base docker.io/golang:alpine \
-		sh -c 'apk add bash gcc musl-dev libxkbcommon-dev pkgconf wayland-dev \
+		sh -c 'apk add bash git gcc musl-dev libxkbcommon-dev pkgconf wayland-dev \
```
Comment (leif) [inline Makefile:77]: I see you added git to the alpine image here, was that intentional?

**Review by leif — COMMENTED** [review]
:+1: Assuming we don't have a new katzenpost release before the network upgrade, we should merge this as soon as the upgrade happens.

File: Makefile (hunk 1 above)
Comment (mixmasala) [reply Makefile:77]: I some environments proxy.golang.org blocks network connections so if GOPROXY=direct is set, git is needed.

**Comment by leif** [comment]
superseded by #47

**Pull request closed without merging**